src/lib/edge-functions/registry.mjs

```diff
@@ -513,7 +548,9 @@ export class EdgeFunctionsRegistry {
    * @returns {Promise<void>}
    */
   async #setupWatcherForDirectory(directory) {
+    const ignored = [new RegExp(this.#servePath)]
```

Comment:
We don't want to watch node modules for changes, and those now live in `servePath`.